apps/dash-mapd-demo/README.md

```diff
@@ -0,0 +1,62 @@
+# dash-mapd-demo
+
+## About this app
+
+This Dash app updates real-time visual analytics by constantly querying a remote Omnisci(MapD) SQL Engine. 
+
+## About this dataset
+
+The public dataset used in this app, "flight_2008_7M", includes every single record of US airline flights known by the Bureau of Transportation Statistics in the year 2008. By default, it can be loaded from the `insert_sample_data` script during the Omnisci server installation. A [jupyter notebook](https://github.com/plotly/dash-mapD-demo/blob/dev/flights_data_clean.ipynb) has been included to showcase tabular structure and query results from this dataset.
+
+## Requirements
+
+* Python 3
+* Omnisci server installation [Guide to install Omnisci](https://www.omnisci.com/docs/latest/4_docker.html) 
+
+## How to run this app
+
+To run this app, you will need a self-hosted Omnisci SQL engine running on `localhost:6274` with the default logins. [follow this guide](https://github.com/plotly/dash-mapD-demo/blob/dev/docker/README.md) to install a test database locally by dockerfile.
+
+We suggest you to create a virtual environment for running this app with Python 3. Clone this repository 
+and open your terminal/command prompt in the root folder.
+
+```
+cd dash-mapd-demo
```

Comment:
I believe you have to cd dash-sample-apps/apps/dash-mapd-demo after you clone the repository

Reply:
💯